README.md

```diff
@@ -9,13 +9,15 @@ The app is designed for everyone, from beginners to experienced entrepreneurs an
 
 ## Features
 - **Practice your speech skills**: The user has 3 different practice modes: Sales pitch, public speaking, and interview. The last one works in real-time, meaning the interview questions can change depending on the user's answers thanks to the Chat-GPT API we use.
```

Comment:
I think the sentence "The last one works in real-time, meaning the interview questions can change depending on the user's answers thanks to the Chat-GPT API we use." is irrelevant as every module works in real time.
```suggestion
- **Practice your speech skills**: The user has 3 different practice modes: Sales pitch, public speaking, and interview. The scenario changes depending on the user's answers thanks to the Chat-GPT API we use.
```